test/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -23,7 +24,7 @@ envoy_cc_test(
 
 envoy_cc_test(
     name = "quic_platform_test",
-    srcs = ["quic_platform_test.cc"],
+    srcs = envoy_select_quiche(["quic_platform_test.cc"]),
```

Comment:
@htuch , should we also be applying this to non-test build rules (e.g. those under source/extensions/quic_listeners/quiche/...), or does the omission of QUICHE from source/extensions/extensions_build_config.bzl already suffice for that?

Reply:
I think the latter should suffice, but worth validating.